### DISCUSSION TITLE
Display image error "too large" or similar, if an image exceeds the ASSETS_TRANSFORM_IMAGE_MAX_DIMENSION env var limit

### DISCUSSION THREAD

**Discussion by josias-r** [directus/directus#7983] [Feature Requests]
If an image larger then the ASSETS_TRANSFORM_IMAGE_MAX_DIMENSION env var limit is uploaded, it is displayed the following way:

![image](https://user-images.githubusercontent.com/11424820/132991389-b6bd24dd-8414-4bd1-8c70-797516d09239.png)

Instead of an errored out image, maybe inform the user why the image is not being displayed (since it's too large)

**Comment by rijkvanzanten** [comment]
Wanna open a PR for that? @josias-r 🙂

**Reply by josias-r** [reply]
Will try to do that once I find time ^^

**Reply by rijkvanzanten** [reply]
I think @azrikahar beat you to it  (https://github.com/directus/directus/pull/8069)

**Comment by rijkvanzanten** [comment]
(Created an issue for this: https://github.com/directus/directus/issues/7996)

**Comment by br41nslug** [comment]
both linked issues have been merged since
